galactus.theme

```diff
@@ -196,6 +200,10 @@ function galactus_preprocess_html(&$variables) {
     $variables['#attached']['library'][] = 'galactus/clf-fontawesome';
   }
 
+  // Add a class to define the active CLF theme colour
```

Comment:
Explanation: use the theme colour setting to add a new class to the body tag for use in our local CLF CSS file